crates/atuin-client/src/secrets.rs

```diff
@@ -97,6 +97,21 @@ pub static SECRET_PATTERNS: &[(&str, &str, TestValue)] = &[
     ),
     ("Stripe test key", "sk_test_[0-9a-zA-Z]{24}", TestValue::Single("sk_test_1234567890abcdefghijklmnop")),
     ("Stripe live key", "sk_live_[0-9a-zA-Z]{24}", TestValue::Single("sk_live_1234567890abcdefghijklmnop")),
+    (
+        "Netlify authentication token",
+        "nf[pcoub]_[0-9a-zA-Z]{36}",
```

Comment:
https://answers.netlify.com/t/change-to-the-netlify-authentication-token-format/106146